### ISSUE TITLE
Can't build dependencies any more

### ISSUE THREAD

**Issue by stefhak** [EricssonResearch/openwebrtc#233]
Following the instructions at https://github.com/EricssonResearch/openwebrtc/wiki/Building-OpenWebRTC (starting from scratch) I get to the step "./build-all.sh -Cbr osx" where it fails with the following (end of) log:
"...
Making install in testsuite
make[1]: Nothing to be done for `install'.
Making install in examples
true
Marking build successful in nettle-2.7.1
clean_source, /Users/eplhak/github/temp/openwebrtc/scripts/dependencies
Cleaning source in gnutls
Cloning into 'gnutls'...
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
./build-gnutls.sh: line 19: cd: gnutls: No such file or directory
fatal: ambiguous argument 'gnutls_3_2_4': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
ERROR: Failed to install sources and configure source git"

**Comment by stefhak** [comment]
I did a new test today (clean build following https://github.com/EricssonResearch/openwebrtc/wiki/Building-OpenWebRTC) and get the same error as above (I saw some activities so I figured this might have been fixed). 
I think it is important that the build guides work (and besides I want to create a PR and for doing that I need a working build to test).

**Comment by superdump** [comment]
We are in the process of finalizing the move to the new build system and fixing issues there is too priority. We'll be done this week/next. You're welcome to try that instead.

**Comment by superdump** [comment]
https://github.com/EricssonResearch/openwebrtc/wiki/Building-OpenWebRTC-with-Cerbero

**Comment by stefhak** [comment]
That one builds!

However, I'm lacking info on how to use the result (i.e. where is the new
./out/x86_64...darwin/bin/openwebrtc daemon run).

**Comment by superdump** [comment]
```
cerbero shell
openwebrtc-daemon
```

**Comment by stefhak** [comment]
I need to expose my full ignorance. How exactly do you start the cerbero
shell?

**Comment by richardmitic** [comment]
In the cerbero directory:
```
./cerbero-uninstalled shell
```
While you're in the shell you can also try ```which openwebrtc-daemon``` to find the location.

**Comment by superdump** [comment]
Sorry, I have an alias from `cerbero` with the path to my cerbero-uninstalled.

**Comment by EvilTrev** [comment]
I followed the cerbero steps, and left it compiling all night... it failed on step 33 of the universal build:

```
ld: library not found for -lgstercolorspace
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [openwebrtc-daemon] Error 1
make[2]: *** [all-recursive] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
Running command 'make'

***** Error running 'package' command:
Recipe 'openwebrtc' failed at the build step 'compile'
```

**Comment by nirbheek** [comment]
@EvilTrev, we don't use ercolorspace in Cerbero anymore. When was the last time you updated your Cerbero git checkout? Could you update and try again? It should continue from where it left off.

**Comment by EvilTrev** [comment]
@nirbheek The last time I pulled the content was a few hours before I started that build, but I've since pulled again today and followed all the steps from the beginning... I'm now stuck with another issue. Step 31/33 for:

```
./cerbero-uninstalled -c config/cross-ios-universal.cbc package openwebrtc
```

Fails with errors surrounding OpenSSL.

```
checking usrsctp.h usability... yes
checking usrsctp.h presence... yes
checking for usrsctp.h... yes
checking for usrsctp_init in -lusrsctp... yes
checking openssl/ssl.h usability... yes
checking openssl/ssl.h presence... yes
checking for openssl/ssl.h... yes
checking for SSL_library_init in -lssl... no
configure: error: You need OpenSSL for the build
```

Is this looking outside the cerbero folder for files on my drive? or something that should be built and included? Is it trying to execute a binary that isn't pathed for? My understanding is that Macs have a version of OpenSSL from the get-go?

**Comment by superdump** [comment]
I finished on the same issue yesterday. I'll look into it and get it fixed. It's going to take a little longer before we manage to fix everything with the full migration to cerbero for all platforms. Bare with us.

We build our own OpenSSL for two reasons:
* We need newer versions than are available on most systems to main compatibility due to changes in other browsers. Firefox recently made changes to only support certain types of crypto algorithms for security reasons and this is only available in OpenSSL 1.0.2 and later.
* We found that OpenSSL that made use of hardware acceleration on different platforms would produce broken encryption and so we needed to build our own to ensure functional encryption.
